### ISSUE TITLE
Run script in Docker image on a schedule

### ISSUE THREAD

**Issue by tehniemer** [mza921/Plex-Auto-Collections#67]
I have an enhancement request for the docker image, keep the container running, but run the script inside on a schedule defined by an environment variable. The environment variable could ether be in CRON format or like this

-e SCRIPTINTERVAL=1h | #s or #m or #h or #d :: s = seconds, m = minutes, h = hours, d = days :: Amount of time between each script run
-- | --

**Comment by mza921** [comment]
I think you can accomplish the same with sticking to cron. I use:
```
0 * * * *docker run -d --rm -v /opt/Plex-Auto-Collections:/config burkasaurusrex/plex-auto-collections -c /config/config-movies.yml
30 * * * *docker run -d --rm -v /opt/Plex-Auto-Collections:/config burkasaurusrex/plex-auto-collections -c /config/config-tv.yml
```

Is there a reason you'd want to keep the container running?

**Comment by burkasaurusrex** [comment]
I tend to agree with @mza921 - much more docker-friendly

**Comment by tehniemer** [comment]
@mza921 @burkasaurusrex  to keep with the spirit of Docker and have the container be completely self-contained and run without interaction by the host machine. It makes migration and/or spinning up a new container easier, you only have to touch one file.

**Comment by mouzzampk2014** [comment]
Hi, I am running this in container too. So currently script is running all the time i.e. if new movie added into Plex then script will run and add the movie into collection? or I need to restart the container? Thank you

**Comment by mouzzampk2014** [comment]
Yup confirmed, script continuously running in the background but definitely doesn't impact on any performance. Thanks

**Comment by burkasaurusrex** [comment]
@mouzzampk2014 the script should only run once when the container is first run and then shutdown - is that not the case?

**Comment by mouzzampk2014** [comment]
@burkasaurusrex apologise for late reply, yes script always running and doesn't need restarting.

**Comment by buroa** [comment]
> @burkasaurusrex apologise for late reply, yes script always running and doesn't need restarting.

https://github.com/mza921/Plex-Auto-Collections/blob/master/app/plex_auto_collections.py

Umm............... no way does it continue to run. The script will process and exit, each and every single time.

**Comment by romprod** [comment]
Hey,

Yes this would be nicer to be able to leave the container running and just have the script run periodically inside the container without having to use crontab or something else to repeatedly re-run new containers.
